regtests/ww3_ufs1.1/info

```diff
@@ -8,7 +8,10 @@
 # hourly  wind speed (gfs_wind.nc) and current fields with 3 hrs interval   #
 # (current.nc).                                                             #
 # POINT: points. The unified point output grid is glo_1deg                  #
-#                                                                           #
+# --------------------------------------------------------------------------#
+# if "ufscoarse" option is selected, a coarser grid with 5 deg. resolution  #
```

Comment:
Is this referring to the "ufscoarse" option in matrix.base?

Reply:
Should this instead be expanded to explain grdset_c ?

Reply:
added